sdk/storage/Azure.Storage.Common/api/Azure.Storage.Common.net6.0.cs

```diff
@@ -20,6 +20,19 @@ public partial class DownloadTransferValidationOptions
         public bool AutoValidateChecksum { get { throw null; } set { } }
         public Azure.Storage.StorageChecksumAlgorithm ChecksumAlgorithm { get { throw null; } set { } }
     }
+    public partial class ExpectContinueOptions
+    {
+        public ExpectContinueOptions() { }
+        public System.TimeSpan Backoff { get { throw null; } set { } }
+        public long ContentLengthThreshold { get { throw null; } set { } }
+        public Azure.Storage.ExpectContinueOptions.ApplyHeaderMode Mode { get { throw null; } set { } }
+        public enum ApplyHeaderMode
+        {
+            Auto = 0,
+            On = 1,
+            Off = 2,
```

Comment:
`Off` should probably be `=0`.

Reply:
Disagree. 0 should be the default behavior as I understand it, which this is.